pkg/v1/providers/ytt/03_customizations/capabilities/add_capabilities.yaml

```diff
@@ -479,6 +458,10 @@ spec:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
 #@ end
+
+#@ if data.values.DISABLE_CRS_FOR_ADDON_TYPE and "capabilities/capabilities-controller" in data.values.DISABLE_CRS_FOR_ADDON_TYPE:
```

Comment:
This looks unrelated. Is this a bug we are fixing?

Reply:
Added this logic to handle the scenario where the existing management cluster is upgraded to the new version and capabilities CRD still is on v1beta1. In the upgraded cluster we need to upgrade the Capabilities CRD too else it will break